### ISSUE TITLE
Critical Error on loading

### ISSUE THREAD

**Issue by cgplayer2000** [EndlesNights/ambientdoors#8]
I'm getting this error when I update the package.  This just started today.  
![image](https://user-images.githubusercontent.com/10798879/99456924-26a0b980-28f8-11eb-86d6-7890c9ab0d8c.png)

Nothing come up in the console log.  

The door sounds still work.

**Comment by Boefjim** [comment]
Yep got the same here, it seems the manifest file path has changed, uninstalling the module and reinstalling it resolves the problem
